app/views/grids/_notes.html.erb

```diff
@@ -88,25 +80,6 @@
   </p>
 <% end %>
 
-<script>
-(function(){
-  $(".<%= className %>-<%= randomSeed %> .show-all a").click(function(event) {
-    var cName = ".<%= className %>-<%= randomSeed %>";
-    var sName = cName + " tr.extra";
-    $(sName).toggleClass("hide");
-    if ($(sName)[0].classList.contains("hide"))
-      var hiding = true;    
-    var myElement = event.target;    
-    if (event.target.classList.contains("caret")) 
-      myElement = event.target.parentElement;
-    if (!hiding) 
-      myElement.innerHTML = 'Show less <b class="caret" style = "transform: rotate(180deg);"></b>';
-    else
-      myElement.innerHTML = 'Show more <b class="caret"></b>';
-  });
-  //setupGridSorters(".<%= className %>-<%= randomSeed %>"); Should order the items by time. Commented out because it reorders the items each time show more/less is clicked
```

Comment:
@Yavnikaa You can leave this line along with the script tag.

Reply:
Also, the tests are failing because this JavaScript code was removed. What you need to do now is find the lines where the error is coming from and change number `5` to `3`. The failing tests:

```
NodeSharedTest#test_about_ability_of_power_tags_to_exclude_tags_like_[activities:foo!foo1] (3.[91](https://github.com/publiclab/plots2/runs/5783865654?check_suite_focus=true#step:5:91)s)
        Expected: 5
          Actual: 3
        test/unit/node_shared_test.rb:238:in `block in <class:NodeSharedTest>'

 FAIL NodeSharedTest#test_about_ability_of_power_tags_to_exclude_tags_like_[notes:test!awesome] (3.[94](https://github.com/publiclab/plots2/runs/5783865654?check_suite_focus=true#step:5:94)s)
        Expected: 5
          Actual: 3
        test/unit/node_shared_test.rb:206:in `block in <class:NodeSharedTest>'

= FAIL NodeSharedTest#test_about_ability_of_power_tags_to_exclude_tags_like_[questions:foo!answered] (4.00s)
        Expected: 5
          Actual: 3
        test/unit/node_shared_test.rb:226:in `block in <class:NodeSharedTest>'

 FAIL NodeSharedTest#test_about_ability_of_power_tags_to_exclude_tags_like_[questions:foo!foo1] (4.04s)
        Expected: 5
          Actual: 3
        test/unit/node_shared_test.rb:216:in `block in <class:NodeSharedTest>'

 FAIL NodeSharedTest#test_about_ability_of_power_tags_to_exclude_tags_like_[upgrades:foo!foo1] (4.10s)
        Expected: 5
          Actual: 3
        test/unit/node_shared_test.rb:247:in `block in <class:NodeSharedTest>'

= FAIL NodeSharedTest#test_that_NodeShared_can_be_used_to_convert_doubled_short_codes_like_[notes:activity:spectrometer]_into_tables_which_list_notes_with_the_tag_`activity:spectrometer` (4.13s)
        Expected: 5
          Actual: 3
        test/unit/node_shared_test.rb:83:in `block in <class:NodeSharedTest>'

 FAIL NodeSharedTest#test_that_NodeShared_can_be_used_to_convert_short_codes_like_[activities:foo]_into_tables_which_list_activity_notes (4.16s)
        Expected: 5
          Actual: 3
        test/unit/node_shared_test.rb:[105](https://github.com/publiclab/plots2/runs/5783865654?check_suite_focus=true#step:5:105):in `block in <class:NodeSharedTest>'

== FAIL NodeSharedTest#test_that_NodeShared_can_be_used_to_convert_short_codes_like_[notes:foo]_into_tables_which_list_notes (4.31s)
        Expected: 5
          Actual: 3
        test/unit/node_shared_test.rb:43:in `block in <class:NodeSharedTest>'

= FAIL NodeSharedTest#test_that_NodeShared_can_be_used_to_convert_short_codes_like_[questions:foo]_into_tables_which_list_questions (4.37s)
        Expected: 5
          Actual: 3
        test/unit/node_shared_test.rb:93:in `block in <class:NodeSharedTest>'

 FAIL NodeSharedTest#test_that_NodeShared_can_be_used_to_convert_short_codes_like_[upgrades:latest]_into_tables_which_list_upgrade_notes (4.41s)
        Expected: 5
          Actual: 3
        test/unit/node_shared_test.rb:115:in `block in <class:NodeSharedTest>'
```

Reply:
@Yavnikaa I didn't mean you should add the `show more` code again. I meant leave only the script tag and line 99. This is the only JavaScript code that should remain ```
```
<script>
(function(){
  //setupGridSorters(".<%= className %>-<%= randomSeed %>"); Should order the items by time. Commented out because it reorders the items each time show more/less is clicked
})()
</script>
```